SchedulearnBackend/SchedulearnBackend/Controllers/TeamController.cs

```diff
@@ -49,6 +51,14 @@ public async Task<ActionResult<IEnumerable<Team>>> GetAccessibleTeams(int id)
             return await _teamService.GetAccessibleTeams(id);
         }
 
+        // GET: api/Team/5/topics
+        [HttpGet("{id}/topics")]
+        public async Task<ActionResult<IEnumerable<TeamTopics>>> GetTopicsByTeam(int id)
```

Comment:
Perhaps move this to Topic controller?

Reply:
Perhaps it would be more clear to get these topics by manager id, as in, get all topics from all teams managed by this manager. GetTopicsByManager

Reply:
Add extra controller endpoint, that would get topics by ONE team id (no accessible teams)